src/vs/workbench/services/files/node/fileService.ts

```diff
@@ -195,6 +196,111 @@ export class FileService implements IFileService {
 		return this.doResolveContent(resource, options, (stat, enc) => this.resolveFileContent(stat, enc));
 	}
 
+	private static chunkSize = 64 * 1024;
+	private static chunkBuffer = Buffer.allocUnsafe(FileService.chunkSize);
```

Comment:
Why did you do that as a `static`?
As i see you using this `field` within class, and method which takes value of `field` is not `static`.

`private chunkBuffer: type = Buffer.allocUnsafe(FileService.chunkSize);`

And don't forgot to assign type of `field`